bindings/python/iota_sdk/types/context_input.py

```diff
@@ -5,7 +5,12 @@
 from dataclasses import dataclass, field
 from enum import IntEnum
 from typing import Any, Dict, List, TypeAlias, Union
-from iota_sdk.types.common import HexStr, json
+from iota_sdk.types.common import HexStr, IdWithSlotIndex, json
+
+
+class SlotCommitmentId(IdWithSlotIndex):
```

Comment:
Not too sure about this being here...

Reply:
Me neither, but don't know where else it should be

Reply:
Should I put it in common.py?

Reply:
Alex added `bindings/python/iota_sdk/types/slot.py` for the commitment itself, maybe also create this file and he'll add it to it